### PR TITLE
Basic working app with tests

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

File: src/Conticket/ApiBundle/Controller/EventController.php [hunk 1]
```diff
@@ -62,49 +69,52 @@ public function cgetAction($limit, $offset)
      */
     public function getAction($id)
     {
-        return ['event' => $this->getOr404($id)];
```
Comment (abdala) [inline src/Conticket/ApiBundle/Controller/EventController.php]: Why you removed 'event' here and keep 'order' in https://github.com/PHPeste/conticket/pull/26/files#diff-93060620b8575bcef04f8c5ccf5aae58R63?

Comment (abdala) [inline src/Conticket/ApiBundle/Controller/EventController.php]: @malukenho ping

Comment (malukenho) [inline src/Conticket/ApiBundle/Controller/EventController.php]: @abdala I will finish this until tomorrow. I need to allow behat tests on travis too

File: app/AppKernel.php [hunk 2]
```diff
@@ -34,8 +34,7 @@ public function registerBundles()
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new FOS\RestBundle\FOSRestBundle(),
             new JMS\SerializerBundle\JMSSerializerBundle(),
-            new Snc\RedisBundle\SncRedisBundle(),
-            new AppBundle\AppBundle(),
+//            new Snc\RedisBundle\SncRedisBundle(),
```
Comment (dudemelo) [inline app/AppKernel.php]: Por que estamos comentando o SncRedisBundle?

File: app/config/parameters.yml.dist [hunk 3]
```diff
@@ -1,17 +1,618 @@
...
-    database_user:     root
-    database_password: ~
-
-    mailer_transport:  smtp
-    mailer_host:       127.0.0.1
-    mailer_user:       ~
-    mailer_password:   ~
-
-    secret:            ThisTokenIsNotSoSecretChangeIt
+
```
Comment (dudemelo) [inline app/config/parameters.yml.dist]: Esse HTML deveria ficar nesse arquivo mesmo?

**Pull request merged**